lib/block_view.go

```diff
@@ -118,8 +118,8 @@ type UtxoView struct {
 	// Validator mappings
 	ValidatorPKIDToValidatorEntry map[PKID]*ValidatorEntry
 
-	// Global stake across all validators
-	GlobalStakeAmountNanos *uint256.Int
+	// Global active stake across all validators
```

Comment:
can we expound upon this comment? Something like `The global active stake is the sum of all DESO staked to validators who are not jailed.`